src/dumper_thread.c

```diff
@@ -43,7 +43,7 @@ struct BufferedTask {
     long id;                        // ID of the task
     struct cr_XmlStruct res;        // XML for primary, filelists and other
     cr_Package *pkg;                // Package structure
-    int pkg_from_md;                // If true - package structure if from
+    gboolean clean;                 // If false - package structure if from
```

Comment:
This comment now has twice `If false..`. I suppose the second should be changed to `If true..`?

Also on the first line there is `If false - package structure if from` I think that should have been `If false - package structure is from`. It would be nice if we could fix that as well.